### ISSUE TITLE
Button ['OK'] bug

### ISSUE THREAD

**Issue by alexlib** [OpenPTV/openptv-python#45]
the button ['OK'] stopped working on both wx and qt4 in the Directory selection dialog, see mailing list for the discussion

**Comment by jmha273** [comment]
Yes, this does work on Ubuntu (tested on 12.04 64-bit and 14.04 32-bit) - the OK button was removed and all I have to do is close the dialogue box to open the software on the selected working directory.
